lib/readline.js

```diff
@@ -868,6 +864,14 @@ Interface.prototype._ttyWrite = function(s, key) {
         break;
 
       default:
+        if (key.name === 'tab') {
+          // If tab completion enabled, do that...
+          if (typeof this.completer === 'function') {
+            this._tabComplete();
+            break;
+          }
+        }
```

Comment:
Cool! Refactored per @Fishrock123 and added comment per @brendanashworth  to let linter know that its intentional.